This is a minor fixes for the suggested BTS regexp in the manual
Upstream bug: http://bugs.mutt.org/3514

--- a/doc/manual.xml.head
+++ b/doc/manual.xml.head
@@ -3121,7 +3121,7 @@
 </para>
 
 <screen>
-subscribe [0-9]*.*@bugs.debian.org</screen>
+subscribe [0-9]+.*@bugs.debian.org</screen>
 
 <para>
 as it's often sufficient to just give a portion of the list's e-mail
